person.rb

```diff
@@ -1,3 +1,4 @@
+require './corrector'
```

Comment:
- [ ] Good job so far, but you forgot one project requirement for the `Person` class, it should have many rentals too, please add this 😄
    ![imagen](https://user-images.githubusercontent.com/60789317/141346809-c94c62ab-e459-4cd8-9ba6-5db214933c00.png)

Reply:
Sorry about it! I just did that part!